bindings/ios/MEGASdk.mm

```diff
@@ -1497,6 +1497,26 @@ - (void)killSession:(uint64_t)sessionHandle {
     self.megaApi->killSession(sessionHandle);
 }
 
+- (nonnull NSDate *)overquotaDeadlineDate {
+    return [[NSDate alloc] initWithTimeIntervalSince1970:self.megaApi->getOverquotaDeadlineTs()];
+}
+
+- (nonnull NSArray<NSDate *> *)overquotaWarningDateList {
+    MegaIntegerList *warningTimeIntervalList = self.megaApi->getOverquotaWarningsTs();
+
+    int sizeOfWarningTimestamps = warningTimeIntervalList->size();
+    if (sizeOfWarningTimestamps == 0) {
+        return [[NSMutableArray alloc] initWithCapacity:0];
```

Comment:
NSArray? I don't think we need `NSMutableArray` here

Reply:
Yep, will fix this. But the thing is `Jenkins` build failed and I have no idea, what can I do to fix this? Ask someone who is in SDK to figure out?

Reply:
you can add a new commit to check how it goes, I don't think it is the problem in this PR

Reply:
👍